packages/app/src/signIn/SignInScreen.tsx

```diff
@@ -80,6 +85,10 @@ export default function SignInScreen() {
   const [isPendingServerResponse, setPendingServerResponse] = React.useState(
     false,
   );
+  const [
+    shouldDoubleCheckAccountStatus,
+    setShouldDoubleCheckAccountStatus,
+  ] = React.useState<"requiresUserApproval" | "approved" | null>(null);
```

Comment:
Nit: this name implies a boolean but it's really a more complex state. Perhaps `automaticLoginState`?